### PR TITLE
Add support for OpenBSD/amd64

### PR DESCRIPTION
**Pull request by mpfz0r**
This code is based on the FreeBSD version
and implements roughly the same feature set.

### CODE REVIEW

**Comment by shirou** [comment]
Thank you! I will create OpenBSD environment and check this great PR. Please be patient for a few days.

**Comment by shirou** [comment]
Could you add build tag `!openbsd` to `*_fallback.go` under every sub packages ?

File: host/host_openbsd.go [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+	"os"
+	"os/exec"
+	"runtime"
+	"strconv"
+	"strings"
+	"time"
+	"unsafe"
+
+	"github.com/shirou/gopsutil/internal/common"
+)
```
Comment (shirou) [inline host/host_openbsd.go]: Could you add `process` like freebsd?

File: host/host_openbsd.go [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+		ret.Platform = platform
+		ret.PlatformFamily = family
+		ret.PlatformVersion = version
+	}
+	system, role, err := Virtualization()
+	if err == nil {
+		ret.VirtualizationSystem = system
+		ret.VirtualizationRole = role
+	}
+
```
Comment (shirou) [inline host/host_openbsd.go]: Could you add procs like freebsd?

**Comment by shirou** [comment]
Thank you for your very cool contribution. It works on my OpenBSD 6.0 environment. I write some comments, and if you fix these, I will merge very soon!.

**Comment by mpfz0r** [comment]
Oh, I actually had done these changes, but I pushed from the wrong repository.
Anyway, I've updated the commit.
Cheers.

**Comment by shirou** [comment]
Thank you so much !!

**Pull request merged**